### PR TITLE
chore: update license year

### PR DESCRIPTION
**Pull request by Shinigami92**

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Review by xDivisionByZerox — DISMISSED** [review]

**Review by matthewmayer — CHANGES_REQUESTED** [review]
Maybe could remove the copyright statement from the user agent and mersenne files and just leave it in LICENSE so it's only in one place?

**Comment by ST-DDT** [comment]
Or at least our copyright year.

**Comment by Shinigami92** [comment]
I was now shown various options for changing something that would have to be changed once a year and adjusting 3 lines. I would not like to get involved in discussions and waste potential time researching whether and where license years are required and where not. If this is really that important to you, then it's best to open your own PR and close this one if necessary.

File: LICENSE [hunk 1]
```diff
@@ -38,8 +38,8 @@ http://github.com/marak/faker.js/
 
 faker.js was inspired by and has used data definitions from:
 
- * https://github.com/stympy/faker/ - Copyright (c) 2007-2010 Benjamin Curtis
- * http://search.cpan.org/~jasonk/Data-Faker-0.07/ - Copyright 2004-2005 by Jason Kohles
+- https://github.com/stympy/faker/ - Copyright (c) 2007-2010 Benjamin Curtis
+- http://search.cpan.org/~jasonk/Data-Faker-0.07/ - Copyright 2004-2005 by Jason Kohles
```
Comment (Shinigami92) [inline LICENSE]: somehow my IDE formatted this file

```suggestion
 * https://github.com/stympy/faker/ - Copyright (c) 2007-2010 Benjamin Curtis
 * http://search.cpan.org/~jasonk/Data-Faker-0.07/ - Copyright 2004-2005 by Jason Kohles
```

**Comment by matthewmayer** [comment]
I agree it is an annoying discussion BUT copyright is important. Rather than blindly updating the year once a year it's important we understand what the copyright statement actually means and if it applies to specific files or the whole repo.

**Comment by Shinigami92** [comment]
> I agree it is an annoying discussion BUT copyright is important. Rather than blindly updating the year once a year it's important we understand what the copyright statement actually means and if it applies to specific files or the whole repo.

So you suggest to only touch the specific files if their content changes?
And in this PR I should revert the specific files and only bump the **LICENSE** file in _root_?

**Comment by matthewmayer** [comment]
I'd bump the root one and look into the history of the other files to see if there is some special reason why they have seperate copyright statements. If not they should just be removed as they don't serve any purpose.

File: src/internal/mersenne.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import type { Randomizer } from '../randomizer';
 
 /**
- * Copyright (c) 2022-2023 Faker
+ * Copyright (c) 2022-2024 Faker
```
Comment (Shinigami92) [inline src/internal/mersenne.ts]:
```suggestion
 * Copyright (c) 2022-2023 Faker
```

File: src/modules/internet/user-agent.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022-2023 Faker
+ * Copyright (c) 2022-2024 Faker
```
Comment (Shinigami92) [inline src/modules/internet/user-agent.ts]:
```suggestion
 * Copyright (c) 2022-2023 Faker
```

**Comment by Shinigami92** [comment]
> I'd bump the root one and look into the history of the other files to see if there is some special reason why they have seperate copyright statements. If not they should just be removed as they don't serve any purpose.

So I reset the specific files and left the other task to someone who has the will to do it

**Review by matthewmayer — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**